Project.toml

```diff
@@ -1,13 +1,20 @@
 name = "LRUCache"
 uuid = "8ac3fa9e-de4c-5943-b1dc-09c6b5f20637"
-version = "1.6.0"
+version = "1.6.1"
 
 [compat]
-julia = "1"
+julia = "≥ 1.9"
```

Comment:
you could use [this strategy](https://pkgdocs.julialang.org/v1/creating-packages/#Transition-from-normal-dependency-to-extension) to maintain support for earlier Julia versions. That would make Serialization a direct dep on pre-1.9 and a weakdep on 1.9+

Reply:
Brilliant, didn't know about this. Thanks!